### PR TITLE
Add request attribute jakarta.servlet.error.query_string

### LINKED ISSUE #26480: Servlet 6.1: A new attribute, jakarta.servlet.error.query_string, has been added to the attributes that must be set on the request as part of an error dispatch.
`https://github.com/jakartaee/servlet/issues/59`

### PR DESCRIPTION
**Pull request by pmd1nh**
fixes https://github.com/OpenLiberty/open-liberty/issues/26480

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.1.internal_fat

**Comment by pmd1nh** [comment]
```
io.openliberty.webcontainer.servlet61.fat.tests.Servlet61DispatcherErrorMethodAttributeTest 2 0 0 0 s 3
 test_Dispatcher_requestErrorMethodAttribute 1 0 0 0 s 1
 test_Dispatcher_requestErrorQueryStringAttribute 1 0 0 0 s 2
```

**Comment by pmd1nh** [comment]
#libby

File: dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebApp.java [hunk 1]
```diff
@@ -4653,6 +4653,7 @@ public RequestDispatcher getErrorPageDispatcher(ServletRequest req, ServletError
         //since Servlet 6.1 
         if (com.ibm.ws.webcontainer.osgi.WebContainer.isServlet61orAbove()) {
             req.setAttribute("jakarta.servlet.error.method", httpServletReq.getMethod());
+            req.setAttribute("jakarta.servlet.error.query_string", httpServletReq.getQueryString());
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebApp.java:4656]: Can we reference the attribute name here using `RequestDispatcher.ERROR_QUERY_STRING` and `RequestDispatcher.ERROR_METHOD`? Or is there a reason we don't do that?

Comment (pmd1nh) [reply dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebApp.java:4656]: no, that string is only available in 6.1 API, so we can't reference it from the base.

**Review by pnicolucci — APPROVED** [review]

**Pull request merged**